cmake/Protobuf.cmake

```diff
@@ -1,5 +1,5 @@
-set(protobuf_SOURCE_DIR ${CMAKE_SOURCE_DIR}/wallet-core/build/protobuf/staging/protobuf-3.9.0)
-set(protobuf_source_dir ${CMAKE_SOURCE_DIR}/wallet-core/build/protobuf/staging/protobuf-3.9.0)
+set(protobuf_SOURCE_DIR ${CMAKE_SOURCE_DIR}/build/local/src/protobuf/protobuf-3.9.0)
+set(protobuf_source_dir ${CMAKE_SOURCE_DIR}/build/local/src/protobuf/protobuf-3.9.0)
```

Comment:
We will need to rollback this before merging, otherwise, the submodule setup will break. In future, we need to make it configurable